Dockerfile

```
@@ -324,6 +330,32 @@ RUN \
     --enable-cross-compile && \
   make -j${nproc}
 
+#install gRPC and protobuf dependicies
+COPY /gRPC /tmp/gRPC
+
+WORKDIR /tmp/gRPC
+RUN \
+git clone https://github.com/grpc/grpc grpc
+
+WORKDIR /tmp/gRPC/grpc
+RUN \
+git submodule update --init && \
+# build gRPC
+bazel build :all && \
+mkdir -p cmake/build
+
+WORKDIR /tmp/gRPC/grpc/cmake/build
+RUN \
+cmake ../.. && \
+make -j${nproc} && \
+make install
```

This can be done as follows:

```suggestion
WORKDIR /tmp/grpc-source
RUN \
    git clone --branch "v1.58.0" --recurse-submodules --depth 1 --shallow-submodules https://github.com/grpc/grpc /tmp/grpc-source

WORKDIR /tmp/grpc-source/cmake/build
RUN cmake -DgRPC_BUILD_TESTS=OFF -DgRPC_INSTALL=ON ../.. && \
    make "-j$(nproc)" && \
    make install
```

and bazel if no longer needed should not be installed.

---

yep, fixed